src/pages/HostsPage.vue

```diff
@@ -15,67 +15,59 @@
                     }"
                 :columns="columns"
                 :form-fields="formFields"
-                create-title="Host"
-                @pagination-changed="setPagination"/>
+                create-title="Host"/>
 </template>
 
 <script>
   // import DataTableTemplate from '../components/common/substance/tables/DataTableTemplate';
   import {date} from 'quasar';
   import FDataTable from '../components/common/molecules/feathers/FDataTable/FDataTable';
   import {capitalize, kebabize} from '../utils';
-  import {makeFindPaginateMixin} from '../';
+  import {useFindPaginate} from '../';
+  import {computed, ref} from 'vue/dist/vue';
```

Comment:
`import {computed, ref} from 'vue/dist/vue';` to `import {computed, ref} from 'vue';`

Reply:
resolved